packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -230,12 +224,13 @@ const ReflectionCard = (props: Props) => {
     }
   }
 
-  const showSpotlight = false
+  const showSpotlight = !__PRODUCTION__
   const showSearch =
     phaseType === 'group' &&
-    !inSpotlight &&
+    !isSpotlightOpen &&
     !isComplete &&
     showSpotlight &&
+    !isDemoRoute() &&
```

Comment:
The demo only has a small number of reflections so the Spotlight isn't necessary although it might still be nice to show it off. If you think it's worth including, I suggest that I create a new issue for the demo and tackle that as the last issue in the Spotlight

Reply:
i think it's worth showing off, but we can save that for a later time